backoffice/src/lib/DeleteBeaconFormValues.ts

```diff
@@ -0,0 +1,3 @@
+export interface DeleteBeaconFormValues {
+  reason: string;
```

Comment:
Does this need to be so heavy having it's own object if it's only a single string?

Reply:
It probably doesn't need to be in it's own file, but it does need to be its own type/object yeah. This is because when using Material Design's form components (Formik) with Typescript, we need to define the structure of the form values being captured as a definite type. See NotesEditing.tsx line 14. I've moved the interface into DeleteBeaconView.tsx